bindings/c/include/molecule_builder.h

```diff
@@ -262,15 +262,13 @@ MOLECULE_API_DECORATOR mol_seg_res_t mol_dynvec_builder_finalize(mol_builder_t b
     res.seg.size = MOL_NUM_T_SIZE + builder.number_used + builder.data_used;
     res.seg.ptr = (uint8_t*)malloc(res.seg.size);
     mol_pack_number(res.seg.ptr, &res.seg.size);
-    if (builder.data_used > 0) {
```

Comment:
When N `None` were pushed into a Vector (`N >= 1`), the data_used is still `0`, but N offsets have to be saved.